reference/Counter/CSharp-MVUX/global.json

```diff
@@ -1,6 +1,6 @@
 {
   // To update the version of Uno please update the version of the Uno.Sdk here. See https://aka.platform.uno/upgrade-uno-packages for more information.
   "msbuild-sdks": {
-    "Uno.Sdk": "5.3.90"
+    "Uno.Sdk": "5.4.0-dev.220"
```

Comment:
@eriklimakc, @Kunal22shah we are not using dev versions for the Uno.Samples repos.
@jeromelaban if I am not mistaken we will need to wait for a stable version to be released with the related changes ( @eriklimakc opened https://github.com/unoplatform/uno.extensions-private/issues/208 with the details )

Reply:
Okay, will wait for the new code gen to be a part of a stable release.